bookwyrm/templates/discover/card-header.html

```diff
@@ -4,17 +4,17 @@
 {% with user_path=status.user.local_path username=status.user.display_name book_path=book.local_path book_title=book|book_title %}
 
 {% if status.status_type == 'GeneratedNote' %}
-    {% if status.content == 'wants to read' %}
+    {% if status.content == 'wants to read' or status.content == '<p>wants to read</p>' %}
```

Comment:
This seems quite tied to a specific implementation detail that might be prone to break in the future.

What if we'd add an additional field to the activitypub representation of statuses, something along the lines of `content_md`, so that BookWyrm's parsing side (`to_model`) could look for a the `content_md` field and use that instead of `content` if it is present.
That way we get the best of both worlds, we still render these out for non-BookWyrm instances (i.e. Mastodon), but within BookWyrm handle markdown for remote and local statuses the same way.
Not knowing some the details of the federation piece, this might not be as easy as I think it is 😄

Reply:
It's terrible and super prone to breaking, I agree! But I don't think the most robust fix would be to include the unformatted version of the status. Rather, I think `GeneratedNote` should be sending the reading status change as a field (like how it's done on comments that include reading status changes) with empty content. This PR is just a bandaid.

Reply:
Ah yes, that sounds even better. Which actually also makes me think about how federation of hashtags (as done in #2524) might have a bit of a design issue.